### PR TITLE
Updated layout for Expedition case study

### LINKED ISSUE #394: Expedition Case Study Page
This page is missing and needs to be created. Find the generic cast study design here: https://share.goabstract.com/6fc98f3b-b7ee-4c1f-8ed3-0677e661f087

UPDATED DESIGN:
https://share.goabstract.com/d77ae3eb-bbf2-430e-b11f-8afdcda212a5?mode=design&sha=66c9b3a64b43f748c0e542468d198bf07a84c1bf

### PR DESCRIPTION
**Pull request by pichfl**
closes #394

### CODE REVIEW

**Comment by marcoow** [comment]
@pichfl can you resolve the conflict?

**Comment by marcoow** [comment]
@pichfl can we merge this to `new-simplabs` please? We should never cross-merge branches (`new-simplabs` **is** our `master` in this case).

File: src/ui/components/PageCaseStudyExpedition/template.hbs [hunk 1]
```diff
@@ -1,9 +1,47 @@
...
+      </div>
+      <div class="main-block.sidebar">
+        <Card @label="Client" @src="/assets/images/card/expedition.svg" @alt="Expedition">
+          <p class="typography.body-text">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam vitae aperiam vero, expedita libero voluptatem quisquam id voluptas maxime ea optio quaerat quia quod eos nostrum culpa animi aspernatur distinctio.
+          </p>
+        </Card>
+        <Card @label="Our Expertise" @src="/assets/images/card/elixir.svg">
+          <h3 class="typography.headline-3">
+            <a class="typography.title-link" href="#">
```
Comment (marcoow) [inline src/ui/components/PageCaseStudyExpedition/template.hbs]: this is missing the link

File: src/ui/components/PageCaseStudyExpedition/template.hbs [hunk 2]
```diff
@@ -1,9 +1,47 @@
...
+          The team approached simplabs when they needed help laying the foundation for advanced features in their Elixir and Phoenix based API as well as making sure their Ember.js based client was following best practices.
+        </p>
+      </div>
+      <div class="main-block.sidebar">
+        <Card @label="Client" @src="/assets/images/card/expedition.svg" @alt="Expedition">
+          <p class="typography.body-text">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam vitae aperiam vero, expedita libero voluptatem quisquam id voluptas maxime ea optio quaerat quia quod eos nostrum culpa animi aspernatur distinctio.
+          </p>
+        </Card>
+        <Card @label="Our Expertise" @src="/assets/images/card/elixir.svg">
```
Comment (marcoow) [inline src/ui/components/PageCaseStudyExpedition/template.hbs]: This is showing the Phoenix logo only (despite the file name). I think we don't actually need to show that one at all but should show the Elixir one.

File: src/ui/components/PageCaseStudyExpedition/template.hbs [hunk 3]
```diff
@@ -14,76 +52,30 @@
...
               <h3 class="typography.headline-2">
                 A path into the future
               </h3>
               <p class="typography.body-text">
-                Expedition's in-house team was able to expand on our foundational work, handed over in an open and well-communicated way. The document generated during the initial review phase, along with background information and strategies for taking on each issue, helped them to address relevant issues before they would turn into impediments, keeping their productivity high.
+                Besides working on the core application, simplabs spearheaded several engineering efforts focused around mobile performance, improved load times, server side rendering and localization.
               </p>
+              <ArrowLink @href="#">
+                Learn More
+              </ArrowLink>
```
Comment (marcoow) [inline src/ui/components/PageCaseStudyExpedition/template.hbs]: I'm not sure where this is supposed to link? Can probably be removed?

**Review by marcoow — CHANGES_REQUESTED** [review]
This looks good but there are some minor things we should change. Also, can you take the latest content from `new-simplabs` (https://gallant-hawking-42050a.netlify.com/cases/expedition) plus the first paragraph that#s currently missing there?

_"Expedition is an online travel magazine for global citizens. It lets travellers experience the world's best destinations through the eyes of a local."_

File: src/ui/components/PageCaseStudyExpedition/template.hbs (hunk 2 above)
Comment (pichfl) [reply src/ui/components/PageCaseStudyExpedition/template.hbs]: I put what we had in the mockup. A quick search shows that Elixir itself doesn't provide an SVG based logo.

Either we use the logo that's on the official website or @ghislaineguerin needs to create an SVG version. I'd go with the former.

File: src/ui/components/PageCaseStudyExpedition/template.hbs (hunk 3 above)
Comment (pichfl) [reply src/ui/components/PageCaseStudyExpedition/template.hbs]: Oversight on my end, can be removed 🙂

File: src/ui/components/PageCaseStudyExpedition/template.hbs (hunk 2 above)
Comment (marcoow) [reply src/ui/components/PageCaseStudyExpedition/template.hbs]: We have an SVG version that we can use already: https://gallant-hawking-42050a.netlify.com/cases/expedition

Comment (pichfl) [reply src/ui/components/PageCaseStudyExpedition/template.hbs]: Totally missed that one, compressed and updated.

**Review by marcoow — COMMENTED** [review]
I think leaving the first paragraph in would be good:

_"Expedition is an online travel magazine for global citizens. It lets travellers experience the world's best destinations through the eyes of a local."_

It looks very empty at the top otherwise: 
Bildschirmfoto 2019-05-20 um 16 14 39

**Comment by pichfl** [comment]
Like so?

**Comment by marcoow** [comment]
@pichfl I restructured a bit - let me know what you think.

**Comment by pichfl** [comment]
Looks good 👍

**Review by marcoow — APPROVED** [review]

**Pull request merged**